src/engine/uct/node/mod.rs

```diff
@@ -146,21 +146,38 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
+                if game.winner() == player || !self.config.uct.playout_aftermath { //don't pass if we're losing on the board
```

Comment:
I still don't understand why we check the `playout_aftermath` setting just here and not in the generic `expand()`. In general the code here is confusing as `expand_root()` is only used rarely. If we reuse the subtree we can end up with a pass move as a child of the root node anyway. So the easiest solution would be of course to always allow the pass move.